src/App.tsx

```diff
@@ -15,36 +16,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
+  DEFAULT = '',
+}
+
+type Props = {
```

Comment:
props???
Props - only for components! do not use this name for other types to avoid confusion in the future
```suggestion
type ...... = { // SortParams or SortConfig will be good i think
```